### ISSUE TITLE
blender help and questions 

### ISSUE THREAD

**Issue by choconobo** [choconobo/blender-help#1]
problem 
a rtx 2080 super rendered video in blender much faster than a Radeon 7900xtx environment and blender output settings are the same since i no longer have the rtx 2080 super i cannot collect evidence to show you you'll just have to believe me when i say that with the rtx 2080 super cuda in task manager would run at 100% the cpu wouldn't run 100%  and blender would complete the rendering much faster  the HIP/compute part of the Radeon 7900xtx which is amds equivalent to Nvidia's cuda doesn't seem to do anything while rendering

ps i do know that cuda is less efficient but it should be about 30% and when comparing a rtx 2080 super to a radeon 7900xtx which is newer and considerably more powerful in every way this should make up for the efficiency difference

**Comment by choconobo** [comment]
@deuszex this is my main issue which i hoped zluda would fix it didnt have any effect except made the rendering more stuttery and unstable

**Comment by deuszex** [comment]
Not sure, what are your settings? It works for me like this. Runs about as fast 3080ti too.
![image](https://github.com/choconobo/blender-help/assets/30974986/1579cc18-7ade-4446-8d59-6ca90d3d143e)

**Comment by choconobo** [comment]
same settings blender acts as if there is no gpu and just uses the cpu at 100% this is also the case without zluda by the way

**Comment by deuszex** [comment]
Yeah task manager doesn't really show it well, if you take a look at `compute core` performance, those go up.

**Comment by choconobo** [comment]
not to argue with you but even if your right about task manager not showing the true performance that still doesn't explain how a 7900xtx is 400% slower in blender than a rtx 2080 super

**Comment by choconobo** [comment]
if you want here's a link to a blender project consisting of a mp4 video and mp3 audio its 2 seconds long just as a test if you could render it on your system without changing the output settings and tell me how long it took you to fully render it 

https://drive.google.com/file/d/1MA-_BIl3qPRD9qCqfWciVJDElrYiGw9X/view?usp=drive_link

side note the project is called weirdo because im frustrated with blender being weird lol im not calling you weird or anything and the mp3 file doesn't actually have any sound
